### ISSUE TITLE
Display New Message Form is blocked by browser

### ISSUE THREAD

**Issue by garrit-schroeder** [OfficeDev/office-js-docs-reference#355]
Hello,

the function    Office.context.mailbox.displayNewMessageForm is blocked by browsers because it opens a new window. Allow Popups is a workaround. 

Is there a way around it. Maybe an email compose window on the main page for example? Or maybe a functionality to detect whether popup popups are blocked.

**Comment by ElizabethSamuel-MSFT** [comment]
@exextoc Can you take a look? Thanks.

**Comment by exextoc** [comment]
Currently, displayNewMessageForm failing when the popup blocker is enabled is by-design. The features you suggested are currently not part of the product. We track Outlook add-in feature requests on our [user-voice page](https://officespdev.uservoice.com/forums/224641-general/category/131778-outlook-add-ins). Please add your request there. Feature requests on user-voice are considered when we go through our planning process.

**Comment by ElizabethSamuel-MSFT** [comment]
Thx @exextoc for your response.

@garrit-schroeder Thanks for raising this issue and I hope you do add it to User Voice. I'll close this issue now.

Thanks.
